src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -4167,6 +4167,90 @@ instruct scatterL(indirect mem, vReg src, vReg idx) %{
   ins_pipe(pipe_slow);
 %}
 
+// ------------------------------ Vector Load Gather Partial-------------------------------
+instruct gatherI_partial(vReg dst, indirect mem, vReg idx, pRegGov pTmp, rFlagsReg cr) %{
+  predicate(UseSVE > 0 &&
+            n->as_LoadVectorGather()->memory_size() < MaxVectorSize &&
+           (n->bottom_type()->is_vect()->element_basic_type() == T_INT ||
+            n->bottom_type()->is_vect()->element_basic_type() == T_FLOAT));
```

Comment:
Style nit: add one more space at the beginning of these lines to have better indentation?

Reply:
> Style nit: add one more space at the beginning of these lines to have better indentation?

One more nit: I would suggest to group those gather partial rules together with other gather rules, as well as scatter parts.

Reply:
Sure. I will change that.